cmd/monitor/monitor.go

```diff
@@ -84,7 +89,7 @@ func getChainState(ctx context.Context, ec *ethclient.Client) (*chainState, erro
 
 	cs.PeerCount, err = ec.PeerCount(ctx)
 	if err != nil {
-		// log.Info().Err(err).Msg("Using fake peer count")
+		log.Info().Err(err).Msg("Using fake peer count")
```

Comment:
I'm trying to think, this might be better at the `Debug` level. I think zkEVM doesn't have peers so this will be logged very often.